ceph/nvmegw_cli/execute.py

```diff
@@ -15,9 +15,12 @@ def __init__(self, node, port=5500) -> None:
     def run_nvme_cli(self, action, **kwargs):
         LOG.info(f"NVMe CLI command : {self.name} {action}")
         base_cmd_args = kwargs.get("base_cmd_args", {})
-        base_cmd_args.update(
-            {"server-address": self.node.ip_address, "server-port": self.port}
-        )
+
+        if not base_cmd_args.get("server-address"):
+            base_cmd_args.update({"server-address": self.node.ip_address})
```

Comment:
@sunilkumarn417 but ultimately this will run on all GW nodes, 1/2 or 3/4 nodes would not have logged into cp.stg.icr.io